### DISCUSSION TITLE
[Feature] <Automatically identify geographical location>

### DISCUSSION THREAD

**Discussion by xiagw** [immich-app/immich#9501] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

[Geospy](https://geospy.ai/)
like this feature,
AI identifies the geographical location of the image when no EXIF info 

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is a fun toy, but nowhere near accurate enough to be actually practical.

**Comment by AngelaDMerkel** [comment]
I tried this on a few images and without a very well photographed landmark, it was always incorrect. Google Photos shifted their policy to use only landmark recognition rather than location data for some time and the performance was so poor that they switched back. Somehow I think we're decades rather than years away from this being practical
